vfio/src/vfio_pci.rs

```diff
@@ -588,8 +588,7 @@ impl VfioPciDevice {
 
     /// Map MMIO regions into the guest, and avoid VM exits when the guest tries
     /// to reach those regions.
-    pub fn map_mmio_regions(&mut self, vm: &Arc<VmFd>, mem_slots: u32) -> Result<()> {
-        let mut slot = mem_slots;
+    pub fn map_mmio_regions(&mut self, vm: &Arc<VmFd>, mem_slot: &mut u32) -> Result<()> {
```

Comment:
I think you shouldn't reuse as an  in/out parameter. It would be better to return the new number.

Reply:
Agreed. I fixed that one.